Userland/Libraries/LibJS/Runtime/Temporal/PlainTime.cpp

```diff
@@ -378,12 +335,12 @@ ThrowCompletionOr<PlainTime*> create_temporal_time(GlobalObject& global_object,
     return object;
 }
 
-// 4.5.9 ToTemporalTimeRecord ( temporalTimeLike ), https://tc39.es/proposal-temporal/#sec-temporal-totemporaltimerecord
-ThrowCompletionOr<UnregulatedTemporalTime> to_temporal_time_record(GlobalObject& global_object, Object const& temporal_time_like)
+// 4.5.8 ToTemporalTimeRecord ( temporalTimeLike ), https://tc39.es/proposal-temporal/#sec-temporal-totemporaltimerecord
```

Comment:
Missing `[, completeness]` in the definition to match the signature in https://tc39.es/proposal-temporal/#sec-temporal-totemporaltimerecord